### PR TITLE
Migrate components to ARM64 (except supervisor)

### PR DESCRIPTION
**Pull request by amulobaev**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 # version of Gitpod will be set by the CI job on publishing to https://charts.gitpod.io/
-version:
+version: 0.8.1
 
 hostname: localhost
 certificatesSecret:
   secretName: https-certificates
 
-imagePrefix: gcr.io/gitpod-io/self-hosted/
+imagePrefix: public.ecr.aws/b1o4t7o5/
```
Comment (pavan-tri) [inline chart/values.yaml:11]: This gets overwrriten by prod or stage, dont change this

File: chart/values.yaml [hunk 2]
```diff
@@ -2,13 +2,13 @@
 # Licensed under the MIT License. See License-MIT.txt in the project root for license information.
 
 # version of Gitpod will be set by the CI job on publishing to https://charts.gitpod.io/
-version:
+version: 0.8.1
```
Comment (pavan-tri) [inline chart/values.yaml:5]: Leave it blank, prod or stage will override this

**Review by pavan-tri — CHANGES_REQUESTED** [review]

File: chart/values.yaml (hunk 1 above)
Comment (amulobaev) [reply chart/values.yaml:11]: this value will be updated by CI/CD

File: chart/values.yaml (hunk 2 above)
Comment (amulobaev) [reply chart/values.yaml:5]: this value will be updated by CI/CD

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by vdefeo-xo** [comment]
:white_check_mark: No issues identified in this pull request. 

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR kept your FTAR the same 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by vdefeo-xo** [comment]
Hi, I'm Vinícius Defeo from [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members) and I have just reviewed your PR.
<b>How likely are you to recommend the Code Review service to another developer?</b>

Not at all likely \[ [1](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=1) \] \[ [2](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=2) \] \[ [3](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=3) \] \[ [4](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=4) \] \[ [5](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=5) \] \[ [6](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=6) \] \[ [7](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=7) \] \[ [8](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=8) \] \[ [9](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=9) \] \[ [10](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/7&entry.1163746017=10) \] Very likely

> TIP: Press `Ctrl` (or `⌘` in a Mac) to open the survey in a new tab.

File: components/ide/theia/leeway.Dockerfile [hunk 3]
```diff
@@ -15,30 +15,30 @@ RUN mkdir -p /theia/node/bin \
...
-RUN tar zvxf yarn-v1.15.2.tar.gz
-ENV PATH=$PATH:/yarn-v1.15.2/bin/
+RUN wget https://github.com/yarnpkg/yarn/releases/download/v1.22.10/yarn-v1.22.10.tar.gz
+RUN tar zvxf yarn-v1.22.10.tar.gz
+ENV PATH=$PATH:/yarn-v1.22.10/bin/
 
 # yq - jq for YAML files
 RUN cd /usr/bin \
-    && curl -fsSL https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64 > yq \
+    && curl -fsSL https://github.com/mikefarah/yq/releases/download/v4.6.3/yq_linux_arm64 > yq \
```
[the remark this answers is not in the document]
Comment (amulobaev) [reply components/ide/theia/leeway.Dockerfile]: Fixed

File: components/ide/code/leeway.Dockerfile [hunk 4]
```diff
@@ -18,19 +19,27 @@ RUN mkdir -p /ide/node/bin \
 # rename node executable
 RUN cp /ide/node/bin/node /ide/node/bin/gitpod-node && rm /ide/node/bin/node
 
+FROM ubuntu:18.04 as code_installer
 
-FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12 as code_installer
```
[the remark this answers is not in the document]
Comment (sri-vathsa) [reply components/ide/code/leeway.Dockerfile:22]: The microsoft typescript node based image has an error loading the git context of the repository. As a result all the git related extensions also would not work as expected. For this reason we are now using ubuntu based vscode image.

**Comment by alexander-a-chernyshev** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by alexander-a-chernyshev** [comment]
:white_check_mark: No new issues have been introduced since the previous review.

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR kept your FTAR the same 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by vdefeo-xo** [comment]
:white_check_mark: No new issues have been introduced since the previous review.

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR kept your FTAR the same 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>2</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Review by pavan-tri — APPROVED** [review]
LGTM

**Pull request merged**